macros/utils/data_types/get_data_type_synonyms.sql

```diff
@@ -0,0 +1,107 @@
+{% macro get_data_type_synonyms(exact_data_type) %}
+    {% set result = adapter.dispatch('get_data_type_synonyms','elementary')(exact_data_type) %}
```

Comment:
Maybe do here something like "if result is None return exact_data_type", rather than do an "if" in every implementation later?
(though maybe it's not the best practice to add logic below "adapter.dispatch", not sure, however you feel)

Reply:
In this case I think I'll not add this logic below adapter.dispatch.